src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import { Comments } from '../../types/Comments';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+interface Props {
+  comment: Comments;
```

Comment:
Why comment have a Comments type?
```suggestion
  comment: Comment;
```